src/parser/priest/discipline/modules/azeritetraits/EnduringLuminescence.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import StatTracker from 'parser/core/modules/StatTracker';
+import SPELLS from 'common/SPELLS';
+import DualStatisticBox from 'interface/others/DualStatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+
+import Atonement from '../spells/Atonement';
+import isAtonement from '../core/isAtonement';
+import AtonementDamageSource from '../features/AtonementDamageSource';
+import AtonementApplicationSource from '../features/AtonementApplicationSource';
+
+/**
+ * The healing bonus on Enduring is technically a boost to PWR healing
+ * As such it's 40% stronger than the tooltip due to that
+ */
+const HEALING_MULTIPLIER = 1.4;
+
+class EnduringLuminescence extends Analyzer {
+  static bonusDuration = 1500;
+  atonementBonusHealing = 0;
+  castBonusHealing = 0;
+
+  static dependencies = {
+    atonementDamageSource: AtonementDamageSource,
+    atonementApplicationSource: AtonementApplicationSource,
+    statTracker: StatTracker,
+    atonement: Atonement,
+  };
+
+  constructor(...args) {
+    super(...args);
+
+    this.ranks =
+      this.selectedCombatant.traitRanks(SPELLS.ENDURING_LUMINESCENCE.id) || [];
```

Comment:
I think Zero had a few weeks ago an argument about max-line length stuff and making too much use of the enter-key.
This should be one line